events/schema.md

```diff
@@ -4,30 +4,157 @@ Events published to the bus have a standard schema that allows SDKs and services
-- `correlationId` can be used to track events across services, grouping them by the kind of session that originated them. Could be an article run during ingestion, or a journal page creating a profile. It is optional at this time.
+### Example: downstream search service indexing an article
+
+```
+{
+    "eventId": "448278c4-22d6-11e8-b467-0ed5f89f718b",
+    "happenedAt": "2018-03-08T12:00:00+00:00",
+    "aggregate": {
+        "service": "search",
+        "name": "indexing",
+        "identifier": "18b21fea-5a87-11e8-9c2d-fa7ae01bbebc",
+    },
+    "type": "search-indexing-started",
+    "data": {
+        ... 
+    },
+    "correlationIds": [
+        {
+            "service": "article-store",
+            "name": "article",
+            "identifier": "10627",
+        },
+        {
+            "service": "article-store",
+            "name": "article-version",
+            "identifier": "1",
+        },
+        {
+            "service": "article-store",
+            "name": "article-run",
+            "identifier": "6551f09a-bd23-4f12-a039-f8fa78ef776a",
+        },
+        {
+            "service": "search",
+            "name": "indexing",
+            "identifier": "18b21fea-5a87-11e8-9c2d-fa7ae01bbebc",
+        },
+    ],
+}
+```
+
+### Example: downstream deposit service pushing an article to Crossref
+
+```
+{
+    "eventId": "448278c4-22d6-11e8-b467-0ed5f89f718b",
+    "happenedAt": "2018-03-08T12:00:00+00:00",
+    "aggregate": {
+        "service": "downstream-sample",
+        "name": "deposit",
+        "identifier": "2018-03-08/1", # daily attempts
+    },
+    "type": "downstream-crossref-started",
+    "data": {
+        ... 
+    },
+    "correlationIds": [
+        {
+            "service": "article-store",
+            "name": "article",
+            "identifier": "10627",
+        },
+        {
+            "service": "article-store",
+            "name": "article-version",
+            "identifier": "1",
+        },
+        {
+            "service": "article-store",
+            "name": "article-run",
+            "identifier": "6551f09a-bd23-4f12-a039-f8fa78ef776a",
+        },
+        {
+            "service": "downstream-sample",
+            "name": "deposit",
+            "identifier": "2018-03-08/1",
+        },
+    ],
+}
+```
+
+## Fields
+
+### `eventId`
+
+Unique for all events. It is a UUID version 1 generated with time and a node identifier.
+
+### `happenedAt` 
+
+Identifies when the event has happened (then it can get published, consumed, etc). It follows the `yyyy-mm-ddTHH:MM:SS+00:00` ISO 8601 format, and is expressed in UTC.
+
+### `aggregate`
+
+Identifies the unit of consistency that has produced the event: article run in the bot, article version in the articles store, podcast episode in journal-cms, an article's set of metrics in metrics, a single profile in profiles.
```

Comment:
Not sure where this would be used...

Reply:
Related to https://github.com/libero/walking-skeleton/pull/28/files#r189615065

Reply:
Basically a logical grouping field with stronger guarantees that `data`, which is schemaless

Reply:
(the equivalent to `id` in https://github.com/elifesciences/bus)